ubuntu/python/README.md

```diff
@@ -0,0 +1,15 @@
+# Python Container
+
+This image adds a working Python environment to `databricksruntime/minimal` using virtualenv. For a Conda-based Python environment, an alternative image can be found [here](https://github.com/databricks/containers/tree/master/ubuntu/python-conda).
+
+## Supported Features
+  - Scala Notebooks
+  - Java/Jar jobs
+  - Python Notebooks, Python Jobs
+  - %sh
+
+## Unsupported Features
+  - DBFS FUSE mount (/dbfs)
+  - Ganglia
+  - Spark Submit Jobs
+  - SSH
```

Comment:
todo: need to verify what these two exactly refer to and if they're added in another layer.

Reply:
SSH is tangential, it installs openssh-server.
Spark Submit uses python, so it's worth testing: see "spark submit" here: https://docs.databricks.com/jobs.html

Reply:
Tested spark submit and it works! updated the README.